vfcompositor/app/GUIShell.c

```
@@ -0,0 +1,47 @@
+#include "Compositor.h"
+#include "Shell.h"
+#include "GUIShell.h"
+#include "Keyboard.h"
+#include "Pallete.h"
+#include "Scheduler.h"
+
+static char command_buffer[256];
```

Clang (14.0.6)

[warning] variable 'command_buffer' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)